docs/configuring-playbook-bridge-mautrix-telegram.md

```diff
@@ -12,6 +12,17 @@ matrix_mautrix_telegram_api_id: YOUR_TELEGRAM_APP_ID
 matrix_mautrix_telegram_api_hash: YOUR_TELEGRAM_API_HASH
 ```
 
+**Note**: For architectures different from `amd64` you need to trigger a local rebuild:
```

Comment:
None of this is necessary.

For other bridges we do it automatically. See `group_vars/matrix_servers`.

We can do the same for this one by adding `matrix_mautrix_telegram_container_self_build: "{{ matrix_architecture != 'amd64' }}"` there.